core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/ToBytesSerialisationTest.java

```diff
@@ -27,16 +27,18 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotSame;
 import static org.junit.jupiter.api.Assertions.assertNull;
+import static uk.gov.gchq.gaffer.serialisation.ToBytesSerialiser.EMPTY_BYTES;
 
 public abstract class ToBytesSerialisationTest<T> extends SerialisationTest<T, byte[]> {
 
+    @Test
     @Override
     public void shouldSerialiseNull() throws SerialisationException {
```

Comment:
Test is called "should serialise Null" but your using "EmptyBytes" ?

Reply:
In the default case (and how Gaffer has always worked), serialisers will return `EMPTY_BYTES` when you `serialiseNull()`. Slightly confusing naming but it is called that because it is an empty byte array.
This PR changes it so that specifically `StringSerialiser` will return `NULL_BYTES` when you `serialiseNull()`, with that meaning a byte array with just a 0 byte in it: `new byte[]{0}`.